src/main/java/org/broadinstitute/hellbender/utils/test/TargetsToolsTestUtils.java

```diff
@@ -15,25 +14,21 @@
  */
 public final class TargetsToolsTestUtils {
 
-    private TargetsToolsTestUtils(){} //don't instantiate this
-
-    /**
-     * Returns a {@link File} pointing to the directory that contains the test data.
-     * @return never {@code null}.
-     */
-    public static File getTestDataDir(){
-        return new File(CommandLineProgramTest.getTestDataDir(),"exome");
+    /** Initialize the reference file and dictionary to use for creating intervals. */
+    public TargetsToolsTestUtils(final File referenceFile){
```

Comment:
It should not be the case that you are ever instantiating a utils class.

Reply:
I think if you are trying to avoid calling = new File(getTestDataDir(),"test_reference.fasta"), create a getter which farms out to GATKBaseTest getTestDataDir() to get these files and leave it to external tool authors to avoid using these files.

Reply:
More specifically, refactor the other methods in this class to take the reference dictionary as an argument and have the REFERENCE_FILE just be a string pointer to the file. Utils classes should not have state.

Reply:
This was before an utility class to create `SimpleIntervals` with the reference in the exome test source. It is true that it is not a utils class, but a factory/builder class.

I think that I prefer in this case to hold the dictionary and the refererence to be sure that it correspond to the same one. I renamed to `SimpleIntervalTestFactory` to be clear that it keeps state and it is a factory.

Reply:
This seems like a good solution, you should update the documentation on the file to reflect this change and make it clear to future authors what this class is doing.

Reply:
Can we left this to a different PR? The main problem with this issue is the `BaseTest` and `GATKBaseTest`, and removing this class by adding new methods to `IntervalUtils` will delay even more the reviewing process...

I open an issue to remove it (https://github.com/broadinstitute/gatk/issues/3771) and I promise to prepare a PR for it once this is in...